Library/Formula/entr.rb

```diff
@@ -1,6 +1,6 @@
 class Entr < Formula
   desc "Run arbitrary commands when files change"
-  homepage "http://entrproject.org/"
+  homepage "https://entrproject.org/"
```

Comment:
All URLs are dead, need to use archive.org

Reply:
Fixed in https://github.com/mistydemeo/tigerbrew/commit/8f9488de1af00111ad117f31d389862917660509 - new URL is https://eradman.com/entrproject/